tests/full_wlm/test_generic_orc_launch_batch.py

```diff
@@ -160,19 +169,22 @@ def test_launch_cluster_orc_reconnect(fileutils, wlmutils):
 
     # batch = False to launch on existing allocation
     network_interface = wlmutils.get_test_interface()
-    orc = exp.create_database(6780, db_nodes=3, batch=True, interface=network_interface)
+    orc = exp.create_database(wlmutils.get_test_port(), db_nodes=3, batch=True, interface=network_interface)
     orc.set_path(test_dir)
+
+    test_account = wlmutils.get_test_account()
+    if test_account:
+        orc.batch_settings.set_account(test_account)
```

Comment:
Out of scope for this PR, more-so a note to myself for a potential `BatchSettings` feature:

Iirc, for `SBatchSettings`, if `set_account` receives a false-y value (i.e. empty str or None), a no-op is made.

Is this not the case for all `BatchSettings` derivatives? Should we consider unifying this behavior for script portability??

Reply:
It should be, indeed. I updated LSF, which was the only one with a slightly different behavior (due to the fact that accounts are always needed on LSF). I also updated the base base `BatchSettings` so that it raises an error only if the account is truth-y. This way, if in the future a new batch setting class is implemented and it does not have the concept of an account, not specifying it will be enough to pass tests.